feature-group-definitions/masonry.yml

```diff
@@ -1,11 +1,6 @@
 name: Masonry
+description: Masonry is a type of grid layout where the items on one of the axes are tightly packed together, instead of leaving gaps to align across the other axis.
 spec: https://drafts.csswg.org/css-grid-3/
 status:
   baseline: false
   support: {}
-compat_features:
-  - css.properties.align-tracks
-  - css.properties.grid-template-columns.masonry
-  - css.properties.grid-template-rows.masonry
-  - css.properties.justify-tracks
-  - css.properties.masonry-auto-flow
```

Comment:
This is OK since were making other changes, but don't take it as a license to seek out every such key just yet. 😄 

I think it'll be a lot easier to clean these up when we have the tools to easily determine whether there's a mismatch between web-features and BCD.

Reply:
Makes sense to me, thanks. Especially for cases where we specifically don't want the web-features entry to map 1:1 to what's in BCD.